rasa/plugin.py

```diff
@@ -48,9 +48,17 @@ def refine_cli(
     """Customizable hook for adding CLI commands."""
 
 
+@hookspec(firstresult=True)  # type: ignore[misc]
+def extract_space_from_args(args: argparse.Namespace) -> Dict[Text, Any]:
```

Comment:
I'd name it in a more general way, like `handle_space_args`, as in the future we might want to add extra space-related arguments, not just space name.